c152.tex

```diff
@@ -0,0 +1,267 @@
+\documentclass{article}
+
+\usepackage{calc}
+\usepackage{enumitem}
+\usepackage[paperwidth=6in, paperheight=9in, margin={1cm,1cm}]{geometry}
+\usepackage[utf8]{inputenc}
+\usepackage{multicol}
+\usepackage{textcomp}
+\usepackage[table]{xcolor}
+
+\title{C152 Checklists}
+\author{Jared Ledvina}
+\date{Sept 2017}
+
+\begin{document}
+
+  \include{common}
+
+  \switchheading{C152 GROUND PROCEDURES}
+
+  \begin{multicols*}{3}
+
+    \flightheading{DISPATCH}
+
+    \begin{clitemize}
+      \clitem{Weather}{BRIEF}
+      \clitem{NOTAMs}{CHECK}
+      \clitem{W\&B}{CHECK}
+      \clitem{Perf. Reqs.}{CHECK}
+      \clitem{Squawks}{CHECK}
+    \end{clitemize}
+
+    \flightheading{PREFLIGHT INSPECTION}
+
+    \begin{clitemize}
+      \clitem{Fuel}{DRAIN, CHECK}
+      \clitem{AV1ATE}{CHECK}
+      \clitem{AROW}{CHECK}
+      \clitem{Avionics}{OFF}
+      \clitem{Master}{ON}
+      \clitem{Fuel Gauge}{TRUE}
+      \clitem{Flaps}{EXTEND}
+      \clitem{Electrics}{TEST}
+      \clitem{Electrics}{OFF}
+      \clitem{Master}{OFF}
+      \clitem{Oil}{4.5-5 qts}
+      \clitem{Exterior}{CHECK}
+      \clitem{Ties/Chocks}{REMOVE}
+      \clitem{Baggage}{CHECK}
+      \clitem{Walkaround}{EXECUTE}
+    \end{clitemize}
+
+    \flightheading{BEFORE START}
+
+    \begin{clitemize}
+      \clitem{Preflight}{COMPLETE}
+      \clitem{Passengers}{BRIEF}
+      \clitem{Seats}{ADJUST}
+      \clitem{Seatbelts}{LATCH}
+      \clitem{Fuel Selector}{ON}
+      \clitem{Mixture}{RICH}
+      \clitem{Throttle}{OPEN 1/2"}
+      \clitem{Carb Heat}{COLD}
+      \clitem{Circuit Breakers}{CHECK}
+      \clitem{Electrical Switches}{OFF}
+      \clitem{Beacon}{ON}
+      \clitem{Avionics}{OFF}
+    \end{clitemize}
+    
+    \flightheading{ENGINE START}
+
+    \begin{clitemize}
+      \clitem{Master Switch}{ON}
+      \clitem{Prime}{AS REQ'D}
+      \clitem{Clear Prop}{ANNOUNCE}
+      \clitem{Starter}{ENGAGE}
+      \clitem{Throttle}{1000 RPM}
+      \clitem{Oil Pressure}{CHECK}
+      \clitem{Nav Lights}{AS REQ'D}
+      \clitem{Avionics}{ON}
+    \end{clitemize}
+
+    \switchheading{TAXI AWAY FROM HAZARDS}
```

Comment:
Not sure what to do with this one actually.

Reply:
could make that a note?